### ISSUE TITLE
"lcoals" should be "locals"

### ISSUE THREAD

**Issue by haapamakim** [ahzhezhe/terraform-generator#5]
The code:

```
  tfg.locals({
    foo: 'bar',
  });
```

produces:

```
lcoals {
  foo = "bar"
}
```

**Expected behavior**
```
locals {
  foo = "bar"
}
```

**Comment by ahzhezhe** [comment]
Thanks for pointing out the mistake.
It is fixed in v4.4.0

**Comment by haapamakim** [comment]
> Thanks for pointing out the mistake.
> It is fixed in v4.4.0

Thanks for this great tool!
